src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -306,4 +316,18 @@ public function buildFpm(string $extra_libs, string $use_lld): void
 
         $this->deployBinary(BUILD_TARGET_FPM);
     }
+
+    public function buildEmbed(string $extra_libs, string $use_lld): void
+    {
+        $vars = SystemUtil::makeEnvVarString([
+            'EXTRA_CFLAGS' => '-g -Os -fno-ident ' . implode(' ', array_map(fn ($x) => "-Xcompiler {$x}", $this->tune_c_flags)),
+            'EXTRA_LIBS' => $extra_libs,
+            'EXTRA_LDFLAGS_PROGRAM' => "{$use_lld} -all-static",
+        ]);
+
+        shell()
+            ->cd(SOURCE_PATH . '/php-src')
+            ->exec('sed -i "s|//lib|/lib|g" Makefile')
+            ->exec('make INSTALL_ROOT=' . BUILD_ROOT_PATH . " -j{$this->concurrency} {$vars} install");
```

Comment:
Using this `INSTALL_ROOT` trick may be useful to build other SAPIs too. It would simplify the code and make `static-php-cli` more future-proof.